### DISCUSSION TITLE
Can't connet to api/v1/workflow-events

### DISCUSSION THREAD

**Discussion by SeibertronSS** [argoproj/argo-workflows#8073] [Q&A] [answered]
I deployed an argo workflow and reverse proxyed its ui using nginx. After I submit a helloworld workflow, I can't view its details in WORKFLOW DETAILS. After I click on helloworld workflow nothing is displayed in the page. Initially, the interface `api/v1/workflow-events/?listOptions.resourceVersion` appears as pending. After a long time there will be such an error in the page `Failed to connect to api/v1/workflow-events/?listOptions.resourceVersion`. 
![image](https://user-images.githubusercontent.com/69496864/156722856-59fa76cb-caf8-4043-9b67-4fd85d559517.png)

**Comment by SeibertronSS** [comment, marked as answer]
I found a workaround. When using nginx to proxy argoworkflow, you need to close the nginx cache
```
proxy_buffering off;
```

**Reply by shanisma** [reply]
I also added get this issue after nginx controller upgrade (nginx-ingress-0.14.0 ,	app version: 2.3.0  ) :

Work again with this annotation : 
`nginx.ingress.kubernetes.io/proxy-buffering: "off" `

**Reply by tlmcphillips** [reply]
I'm hitting this issue today, and even with with the above I'm still seeing timeouts. If its still around mind sharing all the annotations on your ingress?

**Reply by rooney96** [reply]
any chance there's  other solution? 
nginx.ingress.kubernetes.io/proxy-buffering: "off" , is not woking for me either....

**Reply by shanisma** [reply]
Hi guys. I get this error  while I used this Ingress Controller https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/ . Now using I'm using https://github.com/kubernetes/ingress-nginx, work much better. You just need to add  default annotations defined https://github.com/argoproj/argoproj-deployments/blob/master/argo-workflows/resources/argo-server-ingress.yaml.

**Reply by rooney96** [reply]
Yeah Im also using this one and also my ingress resource is set with these annotations...
Its working when running it on docker desktop at my local laptop, but in EKS keeps pending...
I have compared between the Nginx config locally and the one in EKS, both of them generated the same config, but for some reason it fails to load the progress of the jobs in EKS....
